helm/thingsboard/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+name: thingsboard
+description: A Helm chart for Thingsboard
+type: application
+version: 0.1.0
+appVersion: 3.3.3
+icon: https://avatars.githubusercontent.com/u/24291394?s=200&v=4
+home: https://github.com/thingsboard/thingsboard-ce-k8s/
+dependencies:
+  - name: postgresql-ha
+    version: 8.4.0
```

Comment:
let's bump to latest v8.5.2.

Reply:
I'll do it. it was latest when I created it